src/config.ts

```diff
@@ -0,0 +1,76 @@
+export const config = {
```

Comment:
to mozna jakos lepiej nazwac bo co to znaczy config. To jest pewnie bardziej form config